components/script/textinput.rs

```diff
@@ -961,9 +1092,9 @@ impl<T: ClipboardProvider> TextInput<T> {
     }
 
     pub fn set_selection_range(&mut self, start: u32, end: u32, direction: SelectionDirection) {
```

Comment:
Should this method take `ByteOffset` parameters instead of `u32`?

Reply:
I did consider this, but was unsure to what extent I should mess with upstream callers. Looks like the only immediate caller is here: https://github.com/tdelacour/servo/blob/master/components/script/dom/textcontrol.rs#L207. Would it be reasonable to just update this call to wrap `start` / `end` in `UTF8Bytes`? Going farther upstream is starts to get a bit messy, but I don't mind threading the type in higher up if you think that would be better in the long run!

Reply:
Ok. Let’s land the changes already here first, then if you’re interested you could work on another PR.